core/vibes/soul/primitives/favorite/index.tsx

```diff
@@ -3,6 +3,7 @@ import * as Toggle from '@radix-ui/react-toggle';
 import { Heart } from '@/vibes/soul/primitives/favorite/heart';
 
 export interface FavoriteProps {
+  title?: string;
```

Comment:
Hmmm... at first I thought this prop was the native `title` attribute when looking at the consumption. Could we change this to `label` or something else instead so it's not to be confused with native html attributes?